routing/routing_integration_tests/routing_test_tools.cpp

```diff
@@ -79,14 +79,20 @@ namespace integration
 
   unique_ptr<CarRouter> CreateCarRouter(Index & index,
                                         storage::CountryInfoGetter const & infoGetter,
-                                        traffic::TrafficCache const & trafficCache)
+                                        traffic::TrafficCache const & trafficCache,
+                                        vector<LocalCountryFile> const & localFiles)
   {
     auto const countryFileGetter = [&infoGetter](m2::PointD const & pt) {
       return infoGetter.GetRegionCountryId(pt);
     };
 
+    shared_ptr<NumMwmIds> numMwmIds = make_shared<NumMwmIds>();
+    for (LocalCountryFile const & f : localFiles)
+      numMwmIds->RegisterFile(f.GetCountryFile());
+
     auto carRouter = make_unique<CarRouter>(index, countryFileGetter,
-                                            SingleMwmRouter::CreateCarRouter(countryFileGetter, nullptr, trafficCache, index));
+                                            SingleMwmRouter::CreateCarRouter(countryFileGetter, numMwmIds,
```

Comment:
Please, add a comment to the CreateCarRouter that numMwmIds should not be null, also, add a CHECK for that.